cranelift/codegen/src/isa/unwind/systemv.rs

```diff
@@ -123,38 +50,33 @@ impl From<gimli::write::CallFrameInstruction> for CallFrameInstruction {
             CallFrameInstruction::Cfa(reg, offset) => Self::Cfa(reg.0, offset),
             CallFrameInstruction::CfaRegister(reg) => Self::CfaRegister(reg.0),
             CallFrameInstruction::CfaOffset(offset) => Self::CfaOffset(offset),
-            CallFrameInstruction::CfaExpression(expr) => Self::CfaExpression(into_raw_expr(expr)),
             CallFrameInstruction::Restore(reg) => Self::Restore(reg.0),
             CallFrameInstruction::Undefined(reg) => Self::Undefined(reg.0),
             CallFrameInstruction::SameValue(reg) => Self::SameValue(reg.0),
             CallFrameInstruction::Offset(reg, offset) => Self::Offset(reg.0, offset),
             CallFrameInstruction::ValOffset(reg, offset) => Self::ValOffset(reg.0, offset),
             CallFrameInstruction::Register(reg1, reg2) => Self::Register(reg1.0, reg2.0),
-            CallFrameInstruction::Expression(reg, expr) => {
-                Self::Expression(reg.0, into_raw_expr(expr))
-            }
-            CallFrameInstruction::ValExpression(reg, expr) => {
-                Self::ValExpression(reg.0, into_raw_expr(expr))
-            }
             CallFrameInstruction::RememberState => Self::RememberState,
             CallFrameInstruction::RestoreState => Self::RestoreState,
             CallFrameInstruction::ArgsSize(size) => Self::ArgsSize(size),
+            _ => {
+                // The unwind do not generated CallFrameInstruction with
```

Comment:
I'm actually not 100% sure who is not generating what, but if I guessed it correctly, then this would clear things up a bit:

```suggestion
                // Cranelift's unwind support does not generate `CallFrameInstruction`s with
```